docs/tutorial-4-create-svc.md

```diff
@@ -69,6 +69,11 @@ Now export the KUBECFG to set the kubectl context:
 export KUBECONFIG=$path-to-your-kubeconfig.yaml
 ```
 
+Check the namespace that you are locates in the *team-demo* namespace using the following command: 
```

Comment:
Ensure that your default namespace is *team-demo*:

Reply:
And can we use proper english? A thing does not locate, a being does ;)
I don't know what is being said here, so I can't offer a suggestion.